go.mod

```diff
@@ -11,9 +11,9 @@ require (
 	github.com/kubernetes-csi/csi-lib-utils v0.14.0
 	github.com/kubernetes-csi/csi-test/v4 v4.4.0
 	google.golang.org/grpc v1.57.0
-	k8s.io/api v0.28.0-rc.0
-	k8s.io/apimachinery v0.28.0-rc.0
-	k8s.io/client-go v0.28.0-rc.0
+	k8s.io/api v0.28.0
+	k8s.io/apimachinery v0.28.0
+	k8s.io/client-go v0.28.0
 	k8s.io/csi-translation-lib v0.28.0-rc.0
```

Comment:
Tag `v0.28.0` is not yet created for [csi-transition-lib](https://pkg.go.dev/k8s.io/csi-translation-lib?tab=versions)

Reply:
@Sneha-at, can you please fix csi-translation-lib version to v0.28.0? The PR is good otherwise.

Reply:
Sure, I thought we are waiting for the fix to add the version on `pkg.go.dev/k8s.io/csi-translation-lib`. I will update the package version here.